rocket_sqlite_oracle/src/db/kraken.rs

```diff
@@ -257,7 +257,7 @@ mod wire {
 
     #[derive(Debug, Clone)]
     pub struct USDPrice {
-        pub millicents: u64,
+        pub exchange_rate: u64,
     }
```

Comment:
🔧 Can we rename this to something like `BtcPriceInUsd` or something that conveys that we're talking about Bitcoin?